physics/radlw_main.F90

```diff
@@ -8854,25 +8854,25 @@ subroutine cldprmc(nlayers, inflag, iceflag, liqflag, cldfmc,     &
                   abscosno(ig) = 0.0_rb
 
                elseif (iceflag .eq. 0) then
-!                  if (radice .lt. 10.0_rb) stop 'ICE RADIUS TOO SMALL'
-                  abscoice(ig) = absice0(1) + absice0(2)/max(radice,10.0_rb)
+                  if (radice .lt. 10.0_rb) stop 'ICE RADIUS TOO SMALL'
+                  abscoice(ig) = absice0(1) + absice0(2)/radice
```

Comment:
This reverts the changes that Greg Thompson and I agreed on in order to make his updated scheme run (there are no limits to the effective radii in his updated scheme).

Reply:
@DomHeinzeller Dom, i guess it is the results of starting form older code in Mike B. NCAR directory. Should I make a new gsl/develop clone and merge my changes into this updated code?